framework/include/base/Assembly.h

```diff
@@ -2,14 +2,11 @@
 /*               DO NOT MODIFY THIS HEADER                      */
 /* MOOSE - Multiphysics Object Oriented Simulation Environment  */
 /*                                                              */
```

Comment:
I think we should change the width to be the 100 line length limit to make the file margins look consistent.